### DISCUSSION TITLE
Collapse the folders in the Search All Fields feature

### DISCUSSION THREAD

**Discussion by mateosantosdev** [zed-industries/zed#19663] [Feature Requests]
Hello!

First of all, thank you for supporting this incredible project!

Is there any way to collapse the files using the Search All Fields feature? An example in VSCode:

![Monosnap zed-main 2024-10-24 08-46-52](https://github.com/user-attachments/assets/cc39e61e-50e9-4f4c-807a-cc6a8019dc78)

**Comment by opsnull** [comment]
I often view project search results in outline panel, it support fold/unfold results, and I found `outline_panel::Open` is a toggled fold/unfold action.

```
{
    "context": "OutlinePanel && not_editing", 
    "bindings": {
      "escape": "menu::Cancel",
      "ctrl-o": ["outline_panel::Open", { "change_selection": false }],
      "enter": ["outline_panel::Open", { "change_selection": false }],
      "tab": ["outline_panel::Open", { "change_selection": false }],
      "ctrl-n": "menu::SelectNext",
      "ctrl-p": "menu::SelectPrev",
      "^": "outline_panel::SelectParent",
    }
  }
```
![image](https://github.com/user-attachments/assets/271da502-2ce7-4ed0-90ad-59c10b505e7e)

**Reply by mateosantosdev** [reply]
Thank you! That workaround could do the trick :D

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
